LAYOUT.md

```diff
@@ -74,6 +73,11 @@ HTML that are used in several page layouts.  These directories have
 underscores at the start of their names to be consistent with Jekyll's
 naming conventions, but the files they contain are for Pandoc.
 
+Most authors should not need to modify any of the support files themselves.
+The only exception is the `_includes/footer.html` file linking the "Source"
+button to [https://github.com/swcarpentry/lesson-template][swc-lesson-template].
```

Comment:
@wking Oops - just looked how it's done in the same file above (but there is was in the code, which makes sense). Thanks, I've updated the PR.